hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -614,6 +614,12 @@
       sw_images: ["//sw/device/tests/sim_dv:spi_host_tx_rx_test:1"]
       en_run_modes: ["sw_test_mode_test_rom"]
     }
+    {
+      name: chip_sw_spi_host_irq
```

Comment:
Which test point in the testplan does this test map too? You will need to update the [testplan HJSON](https://cs.opensource.google/opentitan/opentitan/+/master:hw/top_earlgrey/data/chip_testplan.hjson?q=chip_testplan&ss=opentitan%2Fopentitan) accordingly so there is no unmapped test in the nightly dashboard.

Reply:
Sorry, I shouldn't have committed this change, I added this to capture the waves xcelium. Although the test currently does not need a device in the bus, it will eventually and won't be compatible with the simulation. So I'm reverting this change.